### PR TITLE
[MINOR] Fix typos in dev/* scripts.

### PR DESCRIPTION
**Pull request by ashashwat**
## What changes were proposed in this pull request?

Consistency in style, grammar and removal of extraneous characters.

## How was this patch tested?

Manually as this is a doc change.

### CODE REVIEW

File: dev/lint-python [hunk 1]
```diff
@@ -60,9 +60,9 @@ export "PYLINT_HOME=$PYTHONPATH"
 export "PATH=$PYTHONPATH:$PATH"
 
 # There is no need to write this output to a file
-#+ first, but we do so so that the check status can
-#+ be output before the report, like with the
-#+ scalastyle and RAT checks.
```
Comment (HyukjinKwon) [inline dev/lint-python]: Ah, this wasn't from SPARK-23174 tho.

**Comment by HyukjinKwon** [comment]
ok to test

**Comment by HyukjinKwon** [comment]
I would rather suggest to fix these together in the followup already open, 20432 or find out more typos while we are here. 
Can you take another look for other scripts and check if there are similar instances or typos? It'd be nicer if we fix them in batch.

**Comment by ashashwat** [comment]
@HyukjinKwon Let me go ahead and check all the scripts for similar instances or typos.

File: dev/lint-python (hunk 1 above)
Comment (HyukjinKwon) [reply dev/lint-python]: Hmmm .. seems this indicates line continuation and intentional .. ?

**Comment by HyukjinKwon** [comment]
@ashashwat, thanks. Let's update the PR title to something like ..  `[MINOR] Fix typos in dev/* scripts`.

File: dev/lint-python (hunk 1 above)
Comment (ashashwat) [reply dev/lint-python]: Is that so?  We have 100 character limit on a single line according to the style guide.  Maybe all 4 lines could be rearranged?

Comment (HyukjinKwon) [reply dev/lint-python]: Hmm .. @nchammas, some of these seem added by you a long ago. Could you take a look for this one please? I think I am not aware of this. Wonder if we can just remove those.

Comment (nchammas) [reply dev/lint-python]: The `#+` convention is something I picked up from [the Linux Documentation Project](http://tldp.org/LDP/abs/html/here-docs.html#COMMENTBLOCK), if that's what you're referring to. You can safely do away with it and just have the `#`.

It was a "phase". I'm over it now... 😆

Comment (HyukjinKwon) [reply dev/lint-python]: Wow, thank you so much.

**Review by srowen — APPROVED** [review]

**Review by HyukjinKwon — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**